RadixWallet/Clients/FactorSourcesClient/FactorSourcesClient+Interface.swift

```diff
@@ -244,46 +248,48 @@ extension FactorSourcesClient {
 			label: label
 		)
 
-		_ = try await addPrivateHDFactorSource(.init(
-			factorSource: factorSource.embed(),
-			mnemonicWithPasshprase: mnemonicWithPassphrase,
-			saveIntoProfile: true
-		))
+		try await saveFactorSource(factorSource.embed())
```

Comment:
this is `OffDevice` FS, not used (but this is the correct thing to do, not save it into keychain (that was wrong))